src/doc/contrib/src/SUMMARY.md

```diff
@@ -14,6 +14,7 @@
     - [New subcommands](./implementation/subcommands.md)
     - [Console Output](./implementation/console.md)
     - [Filesystem](./implementation/filesystem.md)
+    - [Formatting](./implementation/formatting.md)
```

Comment:
Didn't have a better idea of where to place this.  Seemed too specific to be in the Design section.  Figured that we can always move it in the future if we get better ideas on how we want to handle this.